src/frontend/apps/bbb/DashboardMeetingForm/index.tsx

```diff
@@ -43,40 +71,151 @@ const DashboardMeetingForm = ({ meeting }: DashboardMeetingFormProps) => {
       toast.error(intl.formatMessage(messages.createMeetingFail));
     },
   });
-  const [updatedMeeting, setUpdatedMeeting] = useState(meeting);
+  const updateMeetingMutation = useUpdateMeeting(meeting.id, {
+    onSuccess: () => {
+      toast.success(intl.formatMessage(messages.updateMeetingSuccess));
+      setUpdatedMeetingState({});
+    },
+    onError: () => {
+      toast.error(intl.formatMessage(messages.updateMeetingFail));
+      setUpdatedMeetingState(meeting);
+    },
+  });
+  const [updatedMeetingState, setUpdatedMeetingState] = useState(
+    {} as Partial<Meeting>,
+  );
+  // const [timeoutId, setTimeoutId] = useState(0 as NodeJS.Timeout);
+  const timeoutId = useRef(0 as unknown as NodeJS.Timeout);
```

Comment:
humm, is there any way to remove these casts ?

Reply:
I wish there is… still looking

Reply:
a `number` is not enough ?

This is what we are using here https://github.com/openfun/marsha/blob/abf1e6d2034fd46cc69f29a4ab4a4890244acca6/src/frontend/components/DashboardVideoLivePairing/index.tsx#L78-L88

Reply:
This is the typescript `clearTimeout` definition: `clearTimeout(id?: number): void;`

Reply:
@RVassili  Do you think it's ok like this ? https://github.com/openfun/marsha/pull/1445/commits/6749b2039bc551b4911e5aa518abf2d3208b2b16

Reply:
oh yes, it's ok by appending `window.` before the timeout functions :+1: @lunika

Reply:
By not specifying `window` typescript was using the definition from the node module.

Reply:
https://github.com/openfun/marsha/pull/1445/commits/a994d017573fdcad3b8eeadda06f7d7d3a393603 @lunika @RVassili

Reply:
noice